analyzer/src/main/kotlin/managers/Composer.kt

```diff
@@ -45,7 +45,7 @@ import org.ossreviewtoolkit.model.config.AnalyzerConfiguration
 import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.model.createAndLogIssue
 import org.ossreviewtoolkit.model.orEmpty
-import org.ossreviewtoolkit.model.readJsonFile
+import org.ossreviewtoolkit.model.readTree
```

Comment:
Commit message nit: Remove "a" before "FileFormat" in the title.

Reply:
The "a" is deliberately in there to indicate that `readTree()` is a newly added function, in contrast to "*the* ... `readJsonFile()`" as a existing function.

Reply:
Still seems wrong to me. What about "Replace `Mapper.readJsonFile()` with the new `FileFormat.readTree()`" instead?